src/Components/Components/src/Microsoft.AspNetCore.Components.csproj

```diff
@@ -19,4 +19,18 @@
     <Reference Include="System.ComponentModel.Annotations" />
   </ItemGroup>
 
+  <!-- Pack settings -->
+  <PropertyGroup>
+    <NuspecFile>Microsoft.AspNetCore.Components.nuspec</NuspecFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <NuspecProperty Include="coherentPackageVersion=$(PackageVersion)" />
```

Comment:
@aspnet/build is this the right thing to do? In particular with patches, this may become incorrect no? I recall KoreBuild had a way to get the exact version of a package produced from the build. Do we need to do that?

Reply:
https://github.com/aspnet/AspNetCore/pull/12027#discussion_r302681686